src/lib/pay-request/services/products/client.ts

```diff
@@ -35,7 +35,7 @@ export default class Products extends Client {
      */
     listProductsByType(id: string, type: ProductType): Promise<Product[] | undefined> {
       return client._axios
-        .get(`/v1/api/gateway-account/${id}/products`)
+        .get(`/v1/api/gateway-account/${id}/products`, {params: {type: type}})
```

Comment:
could just be `{params: {type}}`, but this is fine

Reply:
Meh, I’ll change it.